staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -716,9 +720,10 @@ func (c *Cacher) List(ctx context.Context, key string, opts storage.ListOptions,
 	if listVal.Kind() != reflect.Slice {
 		return fmt.Errorf("need a pointer to slice, got %v", listVal.Kind())
 	}
+	var numEvals int
```

Comment:
I'm still not convinced we want to have that for cacher - this will always be equal to the number of fetched.

So I would personally remove that metric.